### PR TITLE
Wg c from Christine

### PR DESCRIPTION
**Pull request by paul-grassi**
Any issues you can edit directly or take it up directly with Christine, since I just merged all her branches into one and sent to my repo.

### CODE REVIEW

**Review by garciamike — APPROVED** [review]
Might still deserve one look but I'm good.

**Comment by paul-grassi** [comment]
@jimfenton I'm good too, but give it a look before I merge.

File: sp800-63c/sec11_examples.md [hunk 1]
```diff
@@ -41,11 +41,11 @@ Authorization statements are beyond the scope of this document and will not be d
...
 
-To authenticate using the session key, the subscriber sends the ticket to the RP along with encrypted data that proves that the subscriber possesses the session key embedded within the Kerberos ticket. Session keys are either used to generate new tickets, or to encrypt and authenticate communications between the subscriber and the RP.
+To authenticate using the session key, the subscriber sends the ticket to the RP, along with encrypted data that proves that the subscriber possesses the session key embedded within the Kerberos ticket. Session keys are either used to generate new tickets, or to encrypt and authenticate communications between the subscriber and the RP.
```
Comment (jricher) [inline sp800-63c/sec11_examples.md:48]: Unnecessary comma

File: sp800-63c/sec4_fal.md [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 *This section is normative.*
 
-This section defines allowable Federation Assurance Levels, or FAL. The FAL describes requirements for how assertions are constructed and secured for a given transaction. These levels can be requested by an RP or required by the configuration of both the RP and the IdP for a given transaction. 
+This section defines allowable Federation Assurance Levels, or FAL. The FAL describes requirements for how assertions are constructed and secured for a given transaction. These levels can be requested by an RP, or required by the configuration of both the RP and the IdP, for a given transaction.
```
Comment (jricher) [inline sp800-63c/sec4_fal.md:7]: Unnecessary and potentially confusing comma.

File: sp800-63c/sec4_fal.md [hunk 3]
```diff
@@ -41,14 +41,14 @@ Government-operated IdPs asserting authentication at AAL2 and all IdPs asserting
 
 ### 4.2 <a name="runtime-decisions"></a>Runtime Decisions
 
-The fact that parties have federated SHALL NOT be interpreted as permission to pass information. The decision of whether an authentication can occur or attributes are passed can be determined by the use of a whitelist, a blacklist, or a runtime decision by an authorized party.
+The fact that parties have federated SHALL NOT be interpreted as permission to pass information. The decision of whether an authentication can occur or attributes passed can be determined by the use of a whitelist, a blacklist, or a runtime decision by an authorized party.
```
Comment (jricher) [inline sp800-63c/sec4_fal.md:44]: restore missing verb "are"

File: sp800-63c/sec5_federation.md [hunk 4]
```diff
@@ -71,21 +71,21 @@ As shown in [Figure 5-3](#63cSec5-Figure3), dynamic registration involves four s
 
 2. Validate. The RP and IdP determine each other's validity. This can be accomplished through keying information, metadata, software statements, or other means.
 
-3. Register RP attributes. The RP sends its attributes to the IdP. The IdP associates those attributes with the RP.
+3. Register RP attributes. The RP sends its attributes to the IdP and the IdP associates those attributes with the RP.
```
Comment (jricher) [inline sp800-63c/sec5_federation.md:74]: Run-on sentence. Separate, add a comma, or add a semicolon.

File: sp800-63c/sec5_federation.md [hunk 5]
```diff
@@ -71,21 +71,21 @@ As shown in [Figure 5-3](#63cSec5-Figure3), dynamic registration involves four s
...
 
 Protocols requiring the transfer of keying information SHALL use a secure method to establish such keying information needed to operate the federated relationship during the registration process, including any shared secrets or public keys. Any symmetric keys used in this relationship SHALL be unique to a pair of federation participants.
 
-IdPs SHALL require runtime decisions (see [Section 4.2](#runtime-decisions)) to be made by an authorized party, such as the subscriber, before releasing user information. An IdP accepting dynamically registered RPs MAY limit the types of attributes and other information made available to such RPs. An RP capable of dynamically registering MAY limit from which IdPs it is willing to accept identity information.
+IdPs SHALL require runtime decisions (see [Section 4.2](#runtime-decisions)) to be made by an authorized party, such as the subscriber, before releasing user information. An IdP accepting dynamically registered RPs MAY limit the types of attributes, and other information, made available to such RPs. An RP capable of dynamically registering MAY limit from which IdPs it is willing to accept identity information.
```
Comment (jricher) [inline sp800-63c/sec5_federation.md:80]: Extraneous commas

File: sp800-63c/sec5_federation.md [hunk 6]
```diff
@@ -125,7 +125,7 @@ Where proxies are used, they function as an IdP on one side and an RP on the oth
 **Figure 5-5 Federation Proxy**
 </div>
 
-A proxied federation model can provide several benefits. Federation proxies can simplify technical integration between the RP and IdP by providing a common interface for integration. Additionally, to the extent a proxy effectively blinds the RP and IdP from each other, it can provide some business confidentiality for organizations that want to guard their subscriber lists from each other. Proxies can also mitigate some of the privacy risks of federation described in [Section 5.2](#privacy-reqs) below.
+A proxied federation model provides several benefits. Federation proxies can simplify technical integration between the RP and IdP by providing a common interface for integration. Additionally, to the extent a proxy effectively blinds the RP and IdP from each other, it can provide some business confidentiality for organizations that want to guard their subscriber lists from each other. Proxies can also mitigate some of the privacy risks of federation described in [Section 5.2](#privacy-reqs) below.
```
Comment (jricher) [inline sp800-63c/sec5_federation.md:128]: "Provides" is too affirmative. Restore "can provide" as it's not a guarantee, it's a potential.

File: sp800-63c/sec7_presentation.md [hunk 7]
```diff
@@ -36,9 +36,9 @@ The assertion reference:
 
 In this model, the assertion itself is requested directly from the IdP by the RP, minimizing chances of interception and manipulation by a third party (including the subscriber themselves).
 
-This method also allows the RP to query the IdP for additional attributes about the subscriber not included in the assertion itself, since back-channel communication can continue to occur after the initial authentication transaction has completed without sending the user back to the IdP. This query occurs using an authorization credential issued alongside the assertion, as described in [Section 6](#assertions).
+This method also allows the RP to query the IdP for additional attributes about the subscriber not included in the assertion itself, since back-channel communication can continue to occur after the initial authentication transaction has been completed, without sending the user back to the IdP. This query occurs using an authorization credential issued alongside the assertion, as described in [Section 6](#assertions).
```
Comment (jricher) [inline sp800-63c/sec7_presentation.md:39]: Extraneous comma

File: sp800-63c/sec9_privacy.md [hunk 8]
```diff
@@ -27,18 +26,18 @@ Federation enables the data exposed to an RP to be minimized, and as a result th
...
+While some proxy structures – typically those that exist primarily to simplify integration - may not offer additional subscriber privacy protection, others offer varying levels of privacy to the subscriber through a range of blinding technologies. Privacy policies may dictate appropriate use of the subscriber attributes and authentication transaction data (e.g., who are ultimately the IdP and RP) by the IdP, RP, and the federation proxy, but technical means such as blinding can increase effectiveness of these policies by making the data more difficult to obtain. As the level of blinding increases, the technical and operational implementation complexity may increase. Proxies need to map transactions to the appropriate parties on either side as well as manage the keys for all parties in the transaction.
```
Comment (jricher) [inline sp800-63c/sec9_privacy.md:38]: If it's going to be turned to dashes, these should be proper emdashes with appropriate spacing, not hyphens.

File: sp800-63c/sec9_privacy.md [hunk 9]
```diff
@@ -27,18 +26,18 @@ Federation enables the data exposed to an RP to be minimized, and as a result th
...
+[Section 5.2](#privacy-reqs) identifies agency requirements to consult their SAOP to determine privacy compliance requirements. It is critical to involve your agency's SAOP in the earliest stages of digital authentication system development in order to assess and mitigate privacy risks and advise the agency on compliance obligations such as whether the federation triggers the Privacy Act of 1974 or the E-Government Act of 2002 requirement to conduct a PIA. For example, if the Agency is serving as an IdP in a federation, it is likely that the Privacy Act requirements will be triggered and require coverage by either a new or existing Privacy Act system of records since credentials would be maintained at the IdP on behalf of any RP it federates with. If, however, the agency is an RP and using a 3rd-party IdP, digital authentication may not trigger the requirements of the Privacy Act, depending on what data passed from the RP is maintained by the agency as the RP (in such instances the agency may have a broader programmatic SORN that covers such data).
```
Comment (jricher) [inline sp800-63c/sec9_privacy.md:29]: Extraneous "in order".

File: sp800-63c/sec9_privacy.md [hunk 10]
```diff
@@ -6,18 +6,17 @@
...
+If an IdP is using a whitelist of RPs as described in [Section 4.2](#runtime-decisions), any RPs on that list are not presented to the subscriber during an authentication transaction. Since the IdP does not provide notice to the subscriber at runtime, the IdP makes its list of whitelisted RPs available to the subscriber so that the subscriber can see which RPs on the whitelist have access to which of their attributes in an authentication transaction. Since IdPs can not share a subscriber's authentication information or attributes with a whitelisted RP outside of an authentication transaction involving the subscriber (see [Section 5.2](#privacy-reqs)), the existence of an RP on a list of IdPs does not indicate that the subscriber's information will be shared. However, if the subscriber logs into any of the whitelisted RPs using the IdP, the attributes indicated will be shared as part of the authentication transaction.
```
Comment (jricher) [inline sp800-63c/sec9_privacy.md:19]: "their" is ambiguous, restore explicit "the subscriber's"

**Review by jricher — COMMENTED** [review]
Even though this has been merged already, there are a number of changes that should be made to correct some of the edits made herein.

**Pull request merged**